octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -117,15 +114,45 @@ def subscribe_feeds(self):
             if self.callback_by_feed.get(channel)
         }
         self.client.add_feed(self.get_feed_name(),
-                             symbols=exchange_symbols,
+                             symbols=self.pairs,
                              channels=self.channels,
                              callbacks=callbacks)
+        for channel in self.channels:
+            self.logger.debug(f"Subscribed to {channel}")
+
+    def _ensure_feed_parameters(self):
+        exchange = cryptofeed_exchanges.EXCHANGE_MAP[self.get_feed_name()](
+            config=self.client.config
+        )
+        self._ensure_symbols(exchange)
+        self._ensure_time_frames(exchange)
+
+    def _ensure_symbols(self, exchange):
+        filtered_pairs = []
+        for pair in self.pairs:
+            if pair in exchange.normalized_symbol_mapping:
+                filtered_pairs.append(pair)
+            else:
+                formatted_pair = pair.replace(self.CRYPTOFEED_DEFAULT_MARKET_SEPARATOR, octobot_commons.MARKET_SEPARATOR)
+                self.logger.error(f"{formatted_pair} pair is not supported by this exchange's websocket")
+        self.pairs = filtered_pairs
+
+    def _ensure_time_frames(self, exchange):
```

Comment:
I think we should rename it filter_exchange_time_frames or adapt_exchange_time_frames